packages/code/src/renderingPlugin.tsx

```diff
@@ -216,11 +325,26 @@ export function TableContent({ lines }: TableContentProps) {
     trimmedLines.pop();
   }
 
+  const lineShouldHighlight = (line: number) => {
+    return highlightLines.some(def => {
+      if (isNumber(def)) {
+        return line === def;
+      }
+
+      if (isArray(def)) {
+        const sortedArr = def.sort();
```

Comment:
we can avoid potentially unexpected side-effects by not sorting in place,`[...def].sort()`

Reply:
👍